static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -254,6 +263,14 @@ define([
             // anything we want to happen after displaying ads
             window.googletag.cmd.push(postDisplay);
 
+            // show sponsorship placeholder if adblock detected
+            var sponsorshipIdFound = isSponsorshipContainerTest();
+            var sponsorshipIdClasses = $(sponsorshipIdFound).attr('class').replace('ad-slot ', '');
```

Comment:
you have to search the DOM for $(sponsorshipIdFound) few times here (any operation in DOM is slow). Maybe assign it to the variable? Also remember to use fastdom